modules/installation-and-upgrade/pages/container-deployment/suma/proxy-deployment-vm-suma.adoc

```diff
@@ -0,0 +1,320 @@
+[[install-vm]]
+= Deploy {productname} {productnumber} Proxy as a Virtual Machine
+
+Images for {productname} {productnumber} are available at link:https://www.suse.com/download/suse-manager/[{productname} {productnumber} VM images].
```

Comment:
We should add an important note here that this preferred way of installation. 

Both for the server and proxy.